templates/report_template.html

```diff
@@ -69,13 +69,12 @@
 							<td style="text-align: center;">{{row.build_days_ago}}</td>
 							<td style="text-align: center;">{{row.compose}}</td>
 							<td style="text-align: center;" bgcolor="#3465a4"><a href="https://www.youtube.com/watch?v=68ugkg9RePc" style="text-decoration: none; color: #000;">{{row.lcb_result}}</a></td>
-							<td style="text-align: center;"><p>N/A</p></td>
 							{% if row.tempest_tests_failed != None %}
 								<td style="text-align: center;"><a href="{{row.tempest_tests_url}}">{{row.tempest_tests_failed}}</a></td>
 							{% else %}
 								<td style="text-align: center;">N/A</td>
 							{% endif %}	
```

Comment:
Do we need this check? When will a successful job have failed tempest tests?

Reply:
It was a precaution. I'll remove